cpp/velox/compute/ArrowTypeUtils.cc

```diff
@@ -53,64 +53,103 @@ std::shared_ptr<arrow::DataType> toArrowTypeFromName(const std::string& type_nam
   // The type name of Array type is like ARRAY<type>.
   std::string arrayType = "ARRAY";
   if (type_name.substr(0, arrayType.length()) == arrayType) {
-    std::size_t start = type_name.find_first_of("<");
-    std::size_t end = type_name.find_last_of(">");
+    std::size_t start = type_name.find_first_of('<');
```

Comment:
@marin-ma  Curious about the essence of this fix is the difference between double quotes and single quotes？

Reply:
It should fix typeName like
a. Multiple inner structs `Row<type,Row<type,type>,Row<type>>`
b. The depth of nested struct is > 1 `Row<Row<Row<type,type>,type>>`

Reply:
Got it. Thanks for you explain. Can you also enable the failed UT in this PR. Because the existing uts may not contain the nested struct use case?